tests/test_moderate.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+import cohere
+
+from utils import get_api_key
+
+co = cohere.Client(get_api_key())
+
+
+class TestModerate(unittest.TestCase):
+
+    def test_success(self):
+        moderations = co.moderate(inputs=['I Love Cohere!'])
+        self.assertEqual(len(moderations), 1)
```

Comment:
Can we test the contents of `moderations` in a non-flaky way? Like doing an example that obviously should fail moderation, or obviously should pass it